docs/system/system-metrics.md

```diff
@@ -977,7 +978,12 @@ This metric is [recommended][MetricRecommended].
 
 | Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
 | -------- | --------------- | ----------- | -------------- | --------- |
-| `system.filesystem.usage` | UpDownCounter | `By` |  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `system.filesystem.usage` | UpDownCounter | `By` | Reports a filesystem's space usage across different states. [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+
+
+**[1]:** The sum over all `system.filesystem.state` values SHOULD equal the total storage
```

Comment:
```suggestion
**[1]:** The sum over all `system.filesystem.usage` values SHOULD equal the total storage
```

Reply:
I followed the same pattern as other's `usage` metrics description, e.g. https://github.com/open-telemetry/semantic-conventions/blob/1dc64eef5215dec963efcb66dfe91360e59ab532/docs/system/system-metrics.md#metric-systemmemoryusage

Should we create a follow-up PR to fix them all? What we want to explain in this note is that the sum of all `system.filesystem.usage` values over the different `system.filesystem.state` attributes should be equal to the limit.

Reply:
> the sum of all system.filesystem.usage values over the different system.filesystem.state attributes

can you fix the description in this PR and then use a follow-up PR to fix the rest?

Reply:
I did a quick search, didn't find elsewhere has the same wrong description. can you list them all?

Reply:
Fixed in https://github.com/open-telemetry/semantic-conventions/pull/1380/commits/db0fd2242196e91258909c7bf958cdd3f44ad7d6

Reply:
> I did a quick search, didn't find elsewhere has the same wrong description. can you list them all?


- https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemmemoryusage
- https://github.com/open-telemetry/semantic-conventions/blob/main/docs/system/system-metrics.md#metric-systemmemorylimit

Reply:
I enabled auto-merge, so the PR will be merged once this discussion is resolved.